### PR TITLE
Create community-meetups.md

### PR DESCRIPTION
**Pull request by thelifeandtimes**
@mikeosborne you game to be the champion on these? or should we make it @thomasurbit <<< is that your github ~dalweb-donfun?

### CODE REVIEW

**Comment by thomas-kroes** [comment]
Yes this is me. If possible I would like to champion it.

**Comment by thelifeandtimes** [comment]
> Yes this is me. If possible I would like to champion it.

ok, made you the champion on these.

File: content/grants/community-meetups.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+grant_id = "B0165"
+champion = "~dalweb-donfun"
+completed = false
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0165&prefill_Grant+Name=Meetup%20Bounty"
++++
+
+# Requirements
+- Meetups should be geographically based; if you have distributed, interest-based communities of which you are a part, check out Urbit Embassies for a way to build digital communities on Mars.
+- Meetups must be publicized and include a public web line with event details (time, date, location), and should be actively promoted via social media, in-person recruitment, flyers, a website, etc. If you need a signal boost, email support@urbit.org.
+- You must host a total of six meetups, with a target once a month.
```
Comment (mikeosborne) [inline content/grants/community-meetups.md]:
```suggestion
- You must host a total of six meetups, with a targeted cadence of once a month.
```

File: content/grants/community-meetups.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Creation and admin of an Urbit group for your locality to which all meetup attendees will be invited, or a fully public group. This group should be actively maintained with info about upcoming events, copies of past presentation materials, or other related meetup content.
```
Comment (mikeosborne) [inline content/grants/community-meetups.md]:
```suggestion
- Creation and admin of an Urbit group for your locality to which all meetup attendees will be invited or a fully public group. This group should be actively maintained with info about upcoming events, copies of past presentation materials, and any other related meetup content.
```

File: content/grants/community-meetups.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- As one of the main goals of meetups is to get people building on Urbit, once your meetup gains some traction, consider looking into proposal development for a grant on which members of your meetup can collaborate!
+- While the specific requirements of the meetup bounty listed below can be summarized as, "run a regular meetup," the overarching goal is to onboard people to Urbit. To judge a meetup as truly successful, the answers to questions like 'How many proposals have a meetup's attendees submitted and completed?' and 'How many people has a meetup onboarded who go on to positively contribute to the culture, technology, or creativity of the network?' should have powerful answers that the meetup organizers are shouting from the rooftops. This does not mean that every meetup is expected to be full of developers, or that a meetup that does not have a huge and ever-growing attendee list would be considered a failure. What it does mean is that meetup organizers and participants should become true champions for Urbit who can't help but share with everyone around them how Urbit can offer solutions to many of the pains of modern networked computing.
```
Comment (mikeosborne) [inline content/grants/community-meetups.md]:
```suggestion
- While the specific requirements of the meetup bounty listed below can be summarized as "run a regular meetup", the overarching goal is to build a local community and onboard people to Urbit. Successful meetups will be judged by metrics like how many proposals have attendees submitted and completed and how many people have been onboarded who go on to positively contribute to the culture, technology, or creativity of the network. This does not mean that every meetup is expected to be full of developers or that a meetup that does not have a huge and ever-growing attendee list would be considered a failure. What it does mean is that meetup organizers and participants should become true champions for Urbit who can't help but share with everyone around them how Urbit can offer solutions to many of the pains of modern networked computing.
```

**Review by mikeosborne — CHANGES_REQUESTED** [review]
I would prefer the term Community Organizer rather than Meetup Organizer. What do you think?

**Comment by thelifeandtimes** [comment]
> I would prefer the term Community Organizer rather than Meetup Organizer. What do you think?

Works for me.

File: content/grants/community-meetups.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- As one of the main goals of meetups is to get people building on Urbit, once your meetup gains some traction, consider looking into proposal development for a grant on which members of your meetup can collaborate!
+- While the specific requirements of the meetup bounty listed below can be summarized as "run a regular meetup", the overarching goal is to build a local community and onboard people to Urbit. Successful meetups will be judged by metrics like how many proposals have attendees submitted and completed and how many people have been onboarded who go on to positively contribute to the culture, technology, or creativity of the network. This does not mean that every meetup is expected to be full of developers or that a meetup that does not have a huge and ever-growing attendee list would be considered a failure. What it does mean is that meetup organizers and participants should become true champions for Urbit who can't help but share with everyone around them how Urbit can offer solutions to many of the pains of modern networked computing.
```
Comment (mikeosborne) [inline content/grants/community-meetups.md]:
```suggestion
- While the specific requirements of the meetup bounty listed below can be summarized as "run a regular meetup", the overarching goal is to build a local community and onboard people to Urbit. Successful meetups will be judged by metrics like how many proposals have attendees submitted and completed and how many people have been onboarded who go on to positively contribute to the culture, technology, or creativity of the network. This does not mean that every meetup is expected to be full of developers or that a meetup that does not have a huge and ever-growing attendee list would be considered a failure. What it does mean is that community organizers and participants should become true champions for Urbit who can't help but share with everyone around them how Urbit can offer solutions to many of the pains of modern networked computing.
```

**Comment by thomas-kroes** [comment]
Yes I like the new phrasing, makes the role in the community more long-term oriented and broader.

**Comment by thelifeandtimes** [comment]
@mikeosborne can you merge this today?

**Review by mikeosborne — APPROVED** [review]

**Pull request merged**